components/dashboard/src/data/prebuilds/prebuild-queries.ts

```diff
@@ -64,21 +64,14 @@ export function useCancelPrebuildMutation() {
     });
 }
 
-export function useTriggerPrebuildQuery(configurationId?: string, gitRef?: string) {
-    // we use useQuery instead of useMutation so that we can get data (prebuildId) directly outside
-    return useQuery<string, Error, string>(
-        ["prebuild-trigger", configurationId, gitRef],
-        async () => {
+export function useTriggerPrebuildMutation(configurationId?: string, gitRef?: string) {
+    return useMutation({
```

Comment:
:open_mouth: Wow, nice find! I'm :100: I reviewed this function this week, and just glanced over this :grimacing: